updatecli/updatecli.d/az-ubuntu-22-04-amd64-images.yml

```diff
@@ -0,0 +1,49 @@
+---
+name: Bump azure `ubuntu 22.04` amd64 image version
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  lastReleaseVersion:
+    kind: shell
+    name: Get the latest `ubuntu 22.04` amd64 image version from Azure
+    spec:
+      command: az login --service-principal -u $AZURE_CLIENT_ID -p $AZURE_CLIENT_SECRET --tenant $AZURE_TENANT_ID > /dev/null && az vm image list --location eastus --publisher Canonical --offer 0001-com-ubuntu-server-jammy --sku 22.04-AMD64 --all --query "[?sku=='22.04-AMD64'].version" -o tsv | sort -u | tail -n 1
+      environments:
+        - name: PATH
+        - name: AZURE_CLIENT_ID
+        - name: AZURE_CLIENT_SECRET
+        - name: AZURE_TENANT_ID
+
+targets:
+  updateVersion:
+    name: Update azure `ubuntu 22.04` amd64 image version in locals
+    sourceid: lastReleaseVersion
+    kind: yaml
+    scmid: default
+    spec:
+      file: ./images-versions.yaml
+      key: $.azure.ubuntu.'22.04'.amd64
+    transformers:
+      - addprefix: '"'
+      - addsuffix: '"'
```

Comment:
Did you test locally? If yes what was the result for your test?

On the CI side, it fails with the following error:

```
ERROR: something went wrong in "target#updateVersion" : something went wrong in target "updateVersion" : "validation error: transformer input is empty"
```